src/Tizen.Applications.AttachPanel/Tizen.Applications.AttachPanel/AttachPanel.cs

```diff
@@ -79,12 +101,10 @@ public AttachPanel(Conformant conformant)
         /// </summary>
         ~AttachPanel()
         {
-            if (isCreationSucceed &&
-                _attachPanel != IntPtr.Zero)
+            if (IsInitialized())
             {
-                Interop.AttachPanel.ErrorCode err = Interop.AttachPanel.DestroyAttachPanel(_attachPanel);
-                CheckException(err);
-                _attachPanel = IntPtr.Zero;
+                Interop.AttachPanel.DestroyAttachPanel(s_attachPanel);
```

Comment:
It internally call EFL API. So, It can be issue run on GC thread. 
(EFL is not thread safe)

Reply:
I have to check this out later with registrating as a JIRA issue.
Thanks